docs/components/motor/gpio/encoded-motor.md

```diff
@@ -32,35 +32,32 @@ Here’s an example configuration:
 {
   "components": [
     {
-      "name": <board_name>,
+      "name": <"your-board-name">,

```

Comment:
I think we'd want the angular brackets inside the quotes, to signify that's the bit to replace

Reply:
👀 If we make this change here we should make it on a ton of other components at this point (like https://docs.viam.com/components/board/). I think it formats ok either way so no strong pref

Reply:
Cool, thanks! That does look less weird